server/service/src/processors/transfer/invoice/test.rs

```diff
@@ -59,6 +59,10 @@ async fn invoice_transfers() {
         r.id = uuid();
     });
 
+    let service_item = inline_init(|r: &mut ItemRow| {
```

Comment:
Added a service line to the transfer tests, hopefully will help catch anything funky going forward 😁